SwaglyricsBot/dev_commands.py

```diff
@@ -22,25 +22,25 @@ async def view_stats(self, ctx):
         total_ram = (psutil.virtual_memory().total >> 30) + 1
         embed = discord.Embed(
             title="Bot Stats",
-            description="Running on a dedicated server with {}GB RAM \n provided by RandomGhost#0666.".format(total_ram))
+            description=f"Running on a dedicated server with {total_ram}GB RAM \n provided by RandomGhost#0666.")
         embed.add_field(name="**__General Info__**", inline=False, value="\u200b")
         embed.add_field(
             name="Owner",
-            value=f"{app_info.owner.name}#{app_info.owner.discriminator}")
-        embed.add_field(name="Latency", value=f"{self.bot.latency*1000:.03f} ms")
+            value=f"{app_info.owner}")
+        embed.add_field(name="Latency", value=f"{self.bot.latency*1000:.03f}ms")
         embed.add_field(name="Guild Count", value=f"{len(self.bot.guilds):,}")
         embed.add_field(name="User Count", value=f"{len(self.bot.users):,}")
         embed.add_field(name="**__Technical Info__**", inline=False, value="\u200b")
         embed.add_field(name="System CPU Usage", value=f"{psutil.cpu_percent():.02f}%")
         embed.add_field(name="System RAM Usage",
                         value=f"{psutil.virtual_memory().used/1048576:.02f} MB")
         embed.add_field(name="System Uptime",
-                        value=str(timedelta(seconds=int(time.time()-psutil.boot_time()))))
+                        value=f'{timedelta(seconds=int(time.time() - psutil.boot_time()))}')
         embed.add_field(name="Bot CPU Usage", value=f"{process.cpu_percent():.02f}%")
-        embed.add_field(name="Bot RAM Usage", value=f"{process.memory_info().rss/1048576:.02f} MB")
+        embed.add_field(name="Bot RAM Usage", value=f"{process.memory_info().rss / 1048576:.02f} MB")
         embed.add_field(name="Bot Uptime",
-                        value=str(timedelta(seconds=int(time.time()-process.create_time()))))
+                        value=f'{timedelta(seconds=int(time.time() - process.create_time()))}')
```

Comment:
these are already strings or wut?

Reply:
Should I change it back to str? I thought it would be more consistent if every string was in quotes

Reply:
Oh I see now, yeah makes sense. I didn't realize you were using an fstring to convert to str. Works ig 🤷‍♂️